src/components/20-molecules/datepicker/index.js

```diff
@@ -729,6 +748,63 @@ class AXADatepicker extends NoShadowDOM {
     // any preexisting error should be cleared
     this.error = null;
   }
+
+  // Switches to the next allowed month, when 'direction' is positive.
+  // Switches to the previous allowed month, when 'direction' is negative.
+  // Has no effect, when there is no next/previous allowed month.
+  showNextMonth(direction = 0) {
+    const MIN = 0;
+    const MAX = 11;
+
+    let month = parseInt(this.month, 10);
+    let year = parseInt(this.year, 10);
+
+    if (direction < 0) {
+      if (month !== MIN) {
+        month -= 1;
+      } else {
+        const prevYear = this.getNextYear(-1);
+
+        if (prevYear < year) {
+          month = MAX;
+          year = prevYear;
+        }
+      }
+    } else if (month !== MAX) {
+      month += 1;
+    } else {
+      const nextYear = this.getNextYear();
+
+      if (nextYear > year) {
+        month = MIN;
+        year = nextYear;
+      }
+    }
+
+    this.initDate(overrideDate(year, month, null, this._date), {
+      tentative: true,
+    });
+  }
+
+  // Returns the next allowed year, when 'direction' is positive.
+  // Returns the previous allowed year, when 'direction' is negative.
+  // Returns the currently selected year, when there is no next/previous allowed year.
+  getNextYear(direction = 0) {
+    const MIN = 0;
+    const MAX = this.allowedyears.length - 1;
+
+    let indexOfYear = this.allowedyears.findIndex(year => year === this.year);
+
+    if (direction < 0) {
```

Comment:
DRYification proposal:

indexOfYear += offset;
const {min, max} = Math;
indexOfYear = min(max(indexOfYear, MIN), MAX); // prevent out-of-bounds navigation